blocks/store/selectors.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import createSelector from 'rememo';
+
+/**
+ * Returns all the available block types.
+ *
+ * @param {Object} state Data state.
+ *
+ * @return {Array} Block Types.
+ */
+export const getBlockTypes = createSelector(
+	( state ) => Object.values( state.blockTypes ),
+	( state ) => state.blockTypes
```

Comment:
Minor: We've not updated yet, but `rememo@3.0.0` requires the second argument function return an array:

https://github.com/aduth/rememo/blob/master/CHANGELOG.md